### PR TITLE
Project react-native-app - Karolin Albinsson

### PR DESCRIPTION
**Pull request by karolinalbinsson**
My reflections on how this week's project turned out:

I tried to keep it really simple this week and aimed only for blue level. Mainly because I did not enjoy working with react native, maybe because of all the strange errors and difficulties  that I faced with installations.

 I am happy that this project is done and that I have got the opportunity to work with native, at least I have learned that I do not like working with it and want to stick with web 👍 

Things I'd like to have clarified or explained in more detail:
- 
 expo: https://expo.io/@karolinalbinsson/project-react-native-app

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
+
+  useEffect(() => {
+    ShakeEvent.addListener(() => {
+      shakePhone();
+    })
+
+    return () => {
+      ShakeEvent.removeListener();
+    }
+  }, []);
```
Comment (HIPPIEKICK) [inline App.js:40]: ⭐

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Really nice that you tried out the shake event! ⭐ You've got some really cool functionality going on and the structure of your project is good. However, your indentation is off in some places so please take a look at that :)

**Comment by karolinalbinsson** [comment]
Hi Matilda! Thank you for the feedback. I have made the changes requested.

File: App.js [hunk 2]
```diff
@@ -1,25 +1,65 @@
...
-`
+  font-size: 30px;
+  color: #00106b;
+  padding: 20px;
+  font-weight: bold;
+`;
+
+const SmallText = styled(Title)`
+font-size: 18px;
+font-weight: bold`;
```
Comment (HIPPIEKICK) [inline App.js:23]: This should also be indented ;)

File: App.js [hunk 3]
```diff
@@ -1,25 +1,65 @@
-import React from 'react'
+import React, { useState , useEffect} from 'react'
 import styled from 'styled-components/native'
 
-const Container = styled.View`
+import { CustomButton } from './components/CustomButton';
+import { AnswerWindow } from './components/AnswerWindow'
+import { ShakeEvent} from './components/ShakeEvent';
```
Comment (HIPPIEKICK) [inline App.js:6]: Semi colons or not - be consistent

**Comment by HIPPIEKICK** [comment]
> Hi Matilda! Thank you for the feedback. I have made the changes requested.

Some indentation issues still, but approving now! Have a look at semi colon usage as well (be consistent).

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**